### PR TITLE
Created new folders with README files for each.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

**Review by MichaelHassan — DISMISSED** [review]
Structure looks good.

**Comment by Ruyzambrano** [comment]
Not even a title for the READMEs?

**Review by Leon-JS02 — DISMISSED** [review]
This seems like a solid baseline to work from.

**Review by Leon-JS02 — APPROVED** [review]
Good changes adding titles.

**Review by MichaelHassan — APPROVED** [review]
Nice Titles.

**Review by paulauus — APPROVED** [review]
Title satisfied.

File: web_scraping/epic_games_scraper/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## Epic Games Scraper
```
Comment (Ruyzambrano) [inline web_scraping/epic_games_scraper/README.md:1]:
```suggestion
# Epic Games Scraper
```

**Pull request merged**